subscription-example/public/adyenSubscriptionImplementation.js

```diff
@@ -0,0 +1,97 @@
+const clientKey = document.getElementById("clientKey").innerHTML;
+
+// Used to finalize a checkout call in case of redirect
+const urlParams = new URLSearchParams(window.location.search);
+const sessionId = urlParams.get('sessionId'); // Unique identifier for the payment session
+const redirectResult = urlParams.get('redirectResult');
+
+async function startTokenization() {
+    const type = document.getElementById("type").innerHTML;
+    try {
+        const response = await callServer("/tokenization/sessions");
```

Comment:
Should we prefix here as well? `/api/tokenization/sessions/` (see comment above)

Reply:
Yes, to match the updated URL in `index.js`